app-text/qpdfview/patches/qpdfview-0.4.18.patchset

```diff
@@ -42,3 +42,156 @@ index 6363c32..dc10a8c 100644
 -- 
 2.30.0
 
+
+From c5d6a0ffa020ce7b75469ef778ef7d1f250e0245 Mon Sep 17 00:00:00 2001
+From: FuRuYa7 <mymail@mydomain.org>
+Date: Fri, 29 Oct 2021 15:03:10 +0900
+Subject: [PATCH 1/2] Fix installation path
+
+---
+ qpdfview.pri | 14 +++++++-------
+ 1 file changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/qpdfview.pri b/qpdfview.pri
+index 6dbd353..6155ab7 100644
+--- a/qpdfview.pri
++++ b/qpdfview.pri
+@@ -1,12 +1,12 @@
+ isEmpty(APPLICATION_VERSION):APPLICATION_VERSION = 0.4.18
+ 
+-isEmpty(TARGET_INSTALL_PATH):TARGET_INSTALL_PATH = /usr/bin
+-isEmpty(PLUGIN_INSTALL_PATH):PLUGIN_INSTALL_PATH = /usr/lib/qpdfview
+-isEmpty(DATA_INSTALL_PATH):DATA_INSTALL_PATH = /usr/share/qpdfview
+-isEmpty(MANUAL_INSTALL_PATH):MANUAL_INSTALL_PATH = /usr/share/man/man1
+-isEmpty(ICON_INSTALL_PATH):ICON_INSTALL_PATH = /usr/share/icons/hicolor/scalable/apps
+-isEmpty(LAUNCHER_INSTALL_PATH):LAUNCHER_INSTALL_PATH = /usr/share/applications
+-isEmpty(APPDATA_INSTALL_PATH):APPDATA_INSTALL_PATH = /usr/share/appdata
++isEmpty(TARGET_INSTALL_PATH):TARGET_INSTALL_PATH = $appsDir/qPDFView
++isEmpty(PLUGIN_INSTALL_PATH):PLUGIN_INSTALL_PATH = $appsDir/qPDFView/lib
++isEmpty(DATA_INSTALL_PATH):DATA_INSTALL_PATH = $appsDir/qPDFView/data
++isEmpty(MANUAL_INSTALL_PATH):MANUAL_INSTALL_PATH = $appsDir/qPDFView/man/man1
++isEmpty(ICON_INSTALL_PATH):ICON_INSTALL_PATH = $appsDir/qPDFView/icons/hicolor/scalable/apps
++isEmpty(LAUNCHER_INSTALL_PATH):LAUNCHER_INSTALL_PATH = $appsDir/qPDFView/applications
++isEmpty(APPDATA_INSTALL_PATH):APPDATA_INSTALL_PATH = $appsDir/qPDFView/appdata
+ 
+ win32:include(qpdfview_win32.pri)
+ os2:include(qpdfview_os2.pri)
+-- 
+2.30.2
+
+
+From b5af672c965a1febb3521ed1875d712804fc6350 Mon Sep 17 00:00:00 2001
+From: FuRuYa7 <mymail@mydomain.org>
+Date: Fri, 29 Oct 2021 15:04:08 +0900
+Subject: [PATCH 2/2] Changed to reflect Japanese translation files
```

Comment:
Please link to the upstream revision : https://bazaar.launchpad.net/~adamreichold/qpdfview/trunk/revision/2093

Reply:
Excuse me. I do not understand. There is no "bazaar" skill.

Reply:
Add this link to the patch message. Thanks